### PR TITLE
Fixed code to reflect standard as ESlint indicated, as well as some b…

### PR DESCRIPTION
**Pull request by Jcorrido**
…ugs in route, app, and hello-server.js

### CODE REVIEW

File: hello-server.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 server.listen(port, (error) => {
-    if (error) return console.log(`Error: ${error}`);
+  if (error) return console.log(`Error: ${error}`)
 
-    console.log(`Server is listening on port ${port}`);
-});
\ No newline at end of file
+  console.log(`Server is listening on port ${port}`)
+})
```
Comment (Jcorrido) [inline hello-server.js:14]: Largely altered inorder to have it conform to standard js format

File: routes/routes.js [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
+  app.post('/handles', (request, response) => {
+    pool.query('INSERT INTO Handle SET ?', request.body, (error, result) => {
+      if (error) throw error
+
+      response.status(201).send('Developer added')
+    })
+  })
+}
+
+module.exports = router
```
Comment (Jcorrido) [inline routes/routes.js:53]: shortened

File: app.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 
 const server = app.listen(port, (error) => {
-    if (error) return console.log(`Error: ${error}`);
-
-    console.log(`Server is listening on port ${server.address().port}`);
-});
+  if (error) return console.log(`Error: ${error}`)
 
+  console.log(`Server is listening on port ${server.address().port}`)
+})
```
Comment (Jcorrido) [inline app.js:16]: changed variables and prevented bugs from propping up

**Review by Jcorrido — COMMENTED** [review]
Made fit standard guidelines and prevented possible bugs from propping up by changing some of the variables

File: hello-server.js (hunk 1 above)
Comment (djames1-hub) [reply hello-server.js:14]: Using template string is a great way to make code more readable.

File: app.js [hunk 4]
```diff
@@ -1,17 +1,16 @@
-const express = require('express');
-const bodyParser = require('body-parser');
-const routes = require('./routes/routes');
-const port = 3002;
-const app = express();
+const express = require('express')
```
Comment (djames1-hub) [inline app.js:1]: Removal of semicolon throughout projects makes code easier to develop. Nice!

**Pull request merged**